packages/libs/pactjs-test-project/package.json

```diff
@@ -13,7 +13,7 @@
   "scripts": {
     "_phase:build": "npm run pactjs:generate:contract:file; tsc --noEmit",
     "_phase:test": "npm run test",
-    "build": "tsc --noEmit",
+    "build": "rushx postinstall && tsc --noEmit",
```

Comment:
I suggest replacing the postinstall script with prebuild. Actually, running `rush install` or `rush update --purge` triggers all postinstall scripts. In the case of a fresh clone, it will fail because the "pactjs" dependency is not built yet
```JSON
 "build": "npm run postinstall && tsc --noEmit",
 "prebuild": "npm run pactjs:generate:contract:file && npm run pactjs:generate:template"
 ```

Reply:
It's a bit awkward that we need to even think about this... The steps are I believe:

1. install deps
2. execute post install script of pactjs
3. execute post install script of pactjs-test-project (and any other depending projects)

However it seems that `pact-test-project` did not have `pactjs` available yet during post install, hence this project's post install fails due to a race condition. Are we not able to help rush understand that `pactjs-test-project` depends on the `pactjs` cli dep to be installed? (tbh I was expecting rush to have solved that)

Reply:
Since this is the only package with a dependency on another package in the monorepo during the postinstall stage, I would replace it with a rebuild

Reply:
`rush build` does not run `prebuild` scripts 🤐

[Issue open since September 2018](https://github.com/microsoft/rushstack/issues/830)